### PR TITLE
feat: deploy backend on GAE

### PR DESCRIPTION
**Pull request by yusufaine**
# Scraped idea of deploying on GAE, doesnt support websocket

Either do the following:

1. Host everything on DO, or
2. Split service into those that require ws and those that dont

<details>
<summary>Old plans</summary>
<br>

Current plan:
* [ ] Deploy with F1, allow scaling with min instance of 1, max instance of 20 until end of project.
* [ ] Reduce to 1 min, 1 max by after demo.

Refer to #90 

</details>

### CODE REVIEW

File: backend/package.json [hunk 1]
```diff
@@ -33,7 +40,10 @@
     "test:watch": "jest --watch",
     "test:cov": "jest --coverage",
     "test:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand",
-    "test:e2e": "jest --config ./test/jest-e2e.json"
+    "test:e2e": "jest --config ./test/jest-e2e.json",
+    "clean:zod:index": "rm -rf src/zod/question* src/zod/topic* && head -1 src/zod/index.ts >> src/zod/tmp.ts && mv src/zod/tmp.ts src/zod/index.ts",
```
Comment (aidanaden) [inline backend/package.json:44]: clean function not required, think our prisma zod generator has been commented out

**Review by aidanaden — CHANGES_REQUESTED** [review]
smol changes otherwise LGTM!!

**Review by ngjunkang — APPROVED** [review]
LGTM

**Review by blaketingyu — APPROVED** [review]
LGTM

**Review by aidanaden — APPROVED** [review]

**Pull request merged**